azure-devops.yml

```diff
@@ -0,0 +1,80 @@
+trigger:
+  batch: true
+  branches:
+    include:
+    - master
+pr:
+  paths:
+    exclude:
+    - README.md
+    - docs/*
+phases:
+- phase: Windows
+  displayName: Windows
+
+  condition: succeeded()
+  queue:
+    name: Hosted VS2017
+  steps:
+  - script: |
+      echo Install fake-cli & GTK#
```

Comment:
I think the `&` is interpreted.
It doesn't fail the build, but an error message is shown in the build logs.
```
Install fake-cli 
'GTK#' is not recognized as an internal or external command,
```

Reply:
good catch 👍